src/components/dashboard/Dashboard.js

```diff
@@ -243,6 +245,15 @@ const Dashboard = props => {
     ],
   }
 
+  useEffect(() => {
+    const debouncedHandleResize = debounce(() => {
```

Comment:
debounce can't be defined here, because it is redefined every time so the debounce has no effect